### PR TITLE
Onboard Kube State Metrics onto CLOMonitor

### PR DESCRIPTION
**Pull request by jescalada**
This pull request is for Kubernetes Security Slam, Task 7:

The following issue contains all the related tasks:
https://github.com/kubernetes/kube-state-metrics/issues/2274

Feel free to make any changes as required!

### CODE REVIEW

File: data/cncf.yaml [hunk 1]
```diff
@@ -1419,6 +1419,20 @@
       check_sets:
         - community
         - code
+- name: kube-state-metrics
+  display_name: Kube State Metrics
+  description: Add-on agent to generate and expose cluster-level metrics. 
+  category: observability
+  logo_url: https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/icon/color/kubernetes-icon-color.svg
+  devstats_url: https://kubestatemetrics.devstats.cncf.io/
```
Comment (mrueg) [inline data/cncf.yaml:1427]: I believe we don't have our own devstats url but it all goes through k8s

CC: @puerco @eddie-knight

File: data/cncf.yaml [hunk 2]
```diff
@@ -1419,6 +1419,20 @@
...
+  accepted_at: "2023-12-15"
+  maturity: sandbox
```
Comment (mrueg) [inline data/cncf.yaml:1429]: Kubernetes is graduated and kube-state-metrics is a subproject. Does this mean it should be graduated as well?

File: data/cncf.yaml [hunk 3]
```diff
@@ -1419,6 +1419,20 @@
       check_sets:
         - community
         - code
+- name: kube-state-metrics
+  display_name: Kube State Metrics
+  description: Add-on agent to generate and expose cluster-level metrics. 
+  category: observability
+  logo_url: https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/icon/color/kubernetes-icon-color.svg
+  devstats_url: https://kubestatemetrics.devstats.cncf.io/
+  accepted_at: "2023-12-15"
```
Comment (mrueg) [inline data/cncf.yaml:1428]: Should this be the date of the first commit to kube-state-metrics repository? @eddie-knight @puerco

**Comment by mrueg** [comment]
Thanks for your contribution! I added a couple of questions, from a project side the rest looks good to me.

**Review by mrueg — APPROVED** [review]

File: data/cncf.yaml [hunk 4]
```diff
@@ -1419,6 +1419,20 @@
       check_sets:
         - community
         - code
+- name: kube-state-metrics
+  display_name: Kube State Metrics
+  description: Add-on agent to generate and expose cluster-level metrics. 
+  category: observability
+  logo_url: https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/icon/color/kubernetes-icon-color.svg
+  devstats_url: https://k8s.devstats.cncf.io/
```
Comment (rexagod) [inline data/cncf.yaml:1427]: This should take folks directly to the KSM dashboard.

```suggestion
  devstats_url: https://k8s.devstats.cncf.io/d/49/github-stats-by-repository?orgId=1&var-period=h24&var-repos=%22kubernetes%2Fkube-state-metrics%22&var-stat=reviewers
```

**Review by rexagod — COMMENTED** [review]
Thank you for working on this! :)

File: data/cncf.yaml (hunk 4 above)
Comment (rexagod) [reply data/cncf.yaml:1427]: (I'm assuming here that the dashboard links are not prone to change and stable, feel free to ignore this if that's not the case)

**Comment by tegioz** [comment]
Thanks @jescalada! Kube State Metrics has been added to CLOMonitor as part of #1385.

**Pull request closed without merging**